### PR TITLE
Add meta tags

### PR DESCRIPTION
**Pull request by JoshSmith**
# What's in this PR?

Added meta tags for dynamic meta tag creation. We can add more to this later, but was a good way to get started on this for future use.

### CODE REVIEW

File: app/routes/application.js [hunk 1]
```diff
@@ -12,6 +13,153 @@ export default Ember.Route.extend(ApplicationRouteMixin, {
...
+        name: 'twitter:title',
+        content: 'Code Corps | Build a better future.'
+      },
+    },
+    {
+      type: 'meta',
+      tagId: 'twitter-url',
+      attrs: {
+        name: 'twitter:url',
+        content: ENV.WEB_BASE_URL
```
Comment (WenInCode) [inline app/routes/application.js]: @JoshSmith should this be the link to the twitter and not to the `WEB_BASE_URL`

Comment (JoshSmith) [inline app/routes/application.js]: This doesn't even look like a legit tag.

**Comment by JoshSmith** [comment]
@WenInCode yeah, I'll remove and we'll use that library as well. Will open an issue.

File: app/routes/application.js [hunk 2]
```diff
@@ -12,6 +13,137 @@ export default Ember.Route.extend(ApplicationRouteMixin, {
...
+        property: 'og:site_name',
+        content: 'Code Corps'
+      },
+    },
+    {
+      type: 'meta',
+      tagId: 'meta-og-title',
+      attrs: {
+        property: 'og:title',
+        content: 'Code Corps | Build a better future.'
```
Comment (WenInCode) [inline app/routes/application.js]: Not sure if this should stay if we are removing the other title, but it's not blocking 💄

Comment (JoshSmith) [inline app/routes/application.js]: `og:title` is different, so it should stay.

**Pull request merged**